mits/templates/mits_applications/game.html

```diff
@@ -0,0 +1,110 @@
+{% extends "mits_base.html" %}
+{% block body %}
+
+<h2>Enter Your Game Info</h2>
+
+<form method="post" action="game" class="form-horizontal" role="form">
+    {{ csrf_token() }}
+    <input type="hidden" name="id" value="{{ game.db_id }}" />
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Name</label>
+        <div class="col-sm-6">
+            <input class="form-control focus" type="text" name="name" value="{{ game.name }}" />
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Promo Blurb</label>
+        <div class="col-sm-6">
+            <input class="form-control focus" type="text" name="promo_blurb" value="{{ game.promo_blurb }}" />
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Your Game's Genre:</label>
+        <div class="col-sm-6">
+            <input class="form-control focus" type="text" name="genre" value="{{ game.genre }}" />
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Description</label>
+        <div class="col-sm-6">
+            <textarea class="form-control" name="description" rows="4">{{ game.description }}</textarea>
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Game's Development Phase</label>
+        <div class="col-sm-6">
+            <select name="phase" class="form-control">
+                {{ options(c.MITS_PHASE_OPTS, game.phase) }}
+            </select>
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Min Recommended Age</label>
+        <div class="col-sm-6">
+            <select name="min_age" class="form-control">
+                {{ options(c.MITS_AGE_OPTS, game.min_age) }}
+            </select>
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Min Required Players</label>
+        <div class="col-sm-6">
+            <select name="min_players" class="form-control">
+                {{ int_options(1, 20, game.min_players) }}
+            </select>
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Max Players</label>
+        <div class="col-sm-6">
+            <select name="max_players" class="form-control">
+                {{ int_options(1, 20, game.max_players) }}
+            </select>
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Do You Own This Game?</label>
+        <div class="col-sm-6">
+            {{ macros.checkbox(game, 'personally_own') }}
+            I or the company I represent own this game.
```

Comment:
I've noticed that none of the labels for any of the checkboxes are clickable. Checkboxes without clickable labels are much harder to click. We've made a concerted effort to make all the labels clickable for each checkbox throughout the codebase.

The `checkbox()` macro accepts a `label` argument which will automatically do this for you:
```python
{{ macros.checkbox(game, 'personally_own', label='I or the company I represent own this game.') }}
```

Reply:
Nice, I've just made that change across all 4 of the forms in this plugin which have checkboxes.

I didn't know about checkbox labels 10 years ago when I started this codebase so there are probably still a bunch of places without them, and I've never gotten into the habit of remembering them when making new forms.  I'll try to make a point of keeping that in mind in the future (in all of my projects and not just this one!).